src/smart-components/user/OrgAdminDropdown.tsx

```diff
@@ -0,0 +1,88 @@
+import { Dropdown, DropdownItem, DropdownList, MenuToggle, MenuToggleElement } from '@patternfly/react-core';
+import React, { useEffect, useState } from 'react';
+import { IntlShape } from 'react-intl';
+import messages from '../../Messages';
+import useChrome from '@redhat-cloud-services/frontend-components/useChrome';
+import { useDispatch } from 'react-redux';
+import { fetchUsers, updateUserIsOrgAdminStatus } from '../../redux/actions/user-actions';
+
+const OrgAdminDropdown: React.FC<{
+  isOrgAdmin: boolean;
+  username: string;
+  intl: IntlShape;
+  id: number | undefined;
+}> = ({ isOrgAdmin, username, intl, id }) => {
+  const { auth, isProd } = useChrome();
+  const [isOpen, setIsOpen] = useState(false);
+  const [isDisabled, setIsDisabled] = useState(false);
+  const [loading, setLoading] = useState(false);
+  const [token, setToken] = useState<string | null>(null);
+  const [accountId, setAccountId] = useState<string | null>(null);
+  const [accountUsername, setAccountUsername] = useState<string | null>(null);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    const getToken = async () => {
+      setAccountId((await auth.getUser())?.identity?.internal?.account_id as string);
+      setAccountUsername((await auth.getUser())?.identity?.user?.username as string);
+      setToken((await auth.getToken()) as string);
+    };
+    getToken();
+  }, [auth]);
+
+  useEffect(() => {
+    if (accountUsername === username) {
+      setIsDisabled(true);
+    }
+  }, [username, accountUsername]);
+
+  const onToggleClick = () => {
+    setIsOpen((prev) => !prev);
+  };
+
+  const onSelect = async (_event: React.MouseEvent<Element, MouseEvent> | undefined, value: string | number | undefined) => {
+    if (loading) return;
+
+    const newStatus = value === 'yes';
```

Comment:
This feels a bit dangerous let's use boolean in string.
```suggestion
    const newStatus = value === 'true';
```

Reply:
Fixed